### PR TITLE
Add wgScratchBlocks4BlockVersion to ManageWiki

### PR DESCRIPTION
**Pull request by R4356th**
Adds $wgScratchBlocks4BlockVersion to ManageWiki.

### CODE REVIEW

File: ManageWikiSettings.php [hunk 1]
```diff
@@ -2617,6 +2617,15 @@
 		'help' => 'When enabled, this shows Popups by default.',
 		'requires' => [],
 	],
+	'wgScratchBlocks4BlockVersion' => [
+		'name' => 'Default Scratch Blocks version',
+		'from' => 'scratchblocks',
+		'type' => 'integer',
```
Comment (Universal-Omega) [inline ManageWikiSettings.php:2623]: This should be float if it's a decimal I think

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: ManageWikiSettings.php (hunk 1 above)
Comment (R4356th) [reply ManageWikiSettings.php:2623]: Both are supported by the extension.

Comment (Universal-Omega) [reply ManageWikiSettings.php:2623]: Then it should be set to just 3 at default.

File: LocalSettings.php [hunk 2]
```diff
@@ -3746,6 +3746,11 @@
 			"*",
 		],
 	],
+	
+	// ScratchBlocks
+	'wgScratchBlocks4BlockVersion' => [
+		'default' => '3.0',
```
Comment (Universal-Omega) [inline LocalSettings.php:3752]: This should be int format or MWS type changed to text and default changed to string.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: ManageWikiSettings.php (hunk 1 above)
Comment (R4356th) [reply ManageWikiSettings.php:2623]: I set this to 3 at first as that is what anybody familiar with the extension should expect to see but this is minor so done.

File: LocalSettings.php (hunk 2 above)
Comment (R4356th) [reply LocalSettings.php]: Done.

File: LocalSettings.php [hunk 3]
```diff
@@ -3746,6 +3746,11 @@
 			"*",
 		],
 	],
+	
+	// ScratchBlocks
+	'wgScratchBlocks4BlockVersion' => [
+		'default' => '3',
```
Comment (Universal-Omega) [inline LocalSettings.php:3752]: Remove the quotes, this just needs to be int it seems as that is what default in MWS is.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: LocalSettings.php (hunk 3 above)
Comment (R4356th) [reply LocalSettings.php]: Done.

**Comment by R4356th** [comment]
Blocked on miraheze/mediawiki#1544.

**Comment by Universal-Omega** [comment]
I updated ScratchBlocks4. Will merge this soon.

**Review by Universal-Omega — APPROVED** [review]

**Pull request merged**